Objects/longobject.c

```diff
@@ -3790,6 +3860,41 @@ l_divmod(PyLongObject *v, PyLongObject *w,
     return 0;
 }
 
+/* Compute
+ *     *pmod = mod(v)
+ * NULL can be passed for pmod, in which case the function
+ * never runs code.  The caller owns a reference to pmod.
+ */
```

Comment:
Offhand, it sounds like passing `NULL` for `pmod` should be an error instead. If there's a good reason for supporting a "don't do anything at all" argument, it would be best to explain the intended purpose.

Reply:
What error should be raised?

Reply:
What kind of message should be raised should it be an error?

Reply:
Error should now be raised.